.env-api.sample

```diff
@@ -0,0 +1,6 @@
+ISSUER_SERVER_URL=localhost:3001
+ISSUER_API_UI_SERVER_PORT=3002
+ISSUER_API_UI_AUTH_USER=user
+ISSUER_API_UI_AUTH_PASSWORD=password
```

Comment:
All instances of credentials in samples should be different otherwise it defeats the purpose of having them separate.

Reply:
E.g.

user-api
password-api

user-issuer
password-issuer

user-ui
password-ui